### PR TITLE
Readme: add 'Translations' section and link to Pootle in 'Development resources'

### PR DESCRIPTION
**Pull request by erciccione**

### CODE REVIEW

**Comment by selsta** [comment]
submodule change?

**Comment by erciccione** [comment]
Don't know how that got it. will remove those changes tomorrow

**Comment by mmbyday** [comment]
Probably used git commit -am "commit message" ? 

The -a will add everything different, such as the submodules folder.

The current way the submodules is updated during the build.sh process, the script always pulls the latest master from upstream/remote, probably not intended as it doesn't make any difference. Only tagged versions of master will change the version.js, which is what determines the embedded monero version.

**Comment by erciccione** [comment]
Yes @mmbyday i didn't check after committing, my bad. Fixed now

File: README.md [hunk 1]
```diff
@@ -55,6 +56,13 @@ There are also several mining pools that kindly donate a portion of their fees,
 
 See [LICENSE](LICENSE).
 
+## Translations
+
+If you speak a second language and would like to help translate the Monero GUI wallet, you can do it using our localization platform, Pootle. You can find it at [translate.getmonero.org](https://translate.getmonero.org). Just sign up, navigate to the 'monero-gui' project, find the language you can help with and suggest your changes or add a new translated strings.
```
Comment (xmrdsc) [inline README.md]:
```
Do you speak a second language and would like to help translate the Monero GUI? Sign up at [translate.getmonero.org](https://translate.getmonero.org), our localization platform. Navigate to the 'monero-gui' project and find the language you can help with. Suggest changes or add a new translated text.
```

Comment (xmrdsc) [reply README.md]: What do you think @erciccione ?

**Comment by xmrdsc** [comment]
submodules included ;-)

**Comment by erciccione** [comment]
-.-"
Fixed now

**Pull request merged**